keps/sig-apps/4368-support-managed-by-for-batch-jobs/README.md

```diff
@@ -362,6 +363,52 @@ We may come up with more validation rules during the implementation phase.
 The API comments to the Job status API fields will be updated to make the contract
 clear.
 
+We also plan to add a validation rule to make sure that the terminal conditions
+(`Failed` and `Complete`) are only added to Job when all pods are terminated.
+For that we plan to follow the approach described [below](#terminating-pods-and-terminal-job-conditions).
+
```

Comment:
Can we at least add 
- https://github.com/kubernetes/kubernetes/issues/125089
- https://github.com/kubernetes/kubernetes/issues/125185

as relevant bugs?

And mention to check them before the GA graduation? They do not need to necessarily block the graduation - just for visibility.

And can you also add any other relevant bugs that you might be aware of?

Reply:
I have mentioned them in the text for validation and in the graduation criteria. I think solving these issues blocks adding the validation rules actually.